### PR TITLE
Alexk topline vitals

### LINKED ISSUE #871: Topline vitals summary
**What needs to be done? Why does it need to be done?**
Note: Please develop on top of the branch `feature/vitals`.

The goal of this task is to create a new react component: `CoreVitalSummaryCard.js`. Place 5 of these cards at the top of `CoreCommunityVitals.js` as in the mocks. You can just hardcode values for now. The API for the component should look like this:
```jsx
CoreVitalSummaryCard.propTypes = {
  title: PropTypes.string.isRequired,
  percentage: PropTypes.number.isRequired,
  status: PropTypes.oneOf("POOR", "NEEDS_IMPROVEMENT", "GOOD", "GREAT").isRequired,
  selected: PropTypes.bool,
  onClick: PropTypes.func,
}
```

For example the following card should be created by this jsx:
```jsx
<CoreVitalSummaryCard
  title="Overall"
  percentage={79}
  status="NEEDS_IMPROVEMENT"
  selected
  />
```
Screen Shot 2021-03-07 at 10 35 08 PM

Here's an example of all the different card states:
Screen Shot 2021-03-07 at 10 48 13 PM
A few things to note:
- From left to right, the values of `status` are: `"NEEDS_IMPROVEMENT", "POOR", "GOOD", "GREAT", "GREAT"`
- When selected (see card #2), the top border color should wrap around the rest of the card
- The third and fourth ones get a special treatment for being at or above 95%
  - This involves the slightly green background and the wreath accent
  - You can skip the wreath for now as we finalize the asset.

You can find specific colors, sizing, and general styling in these mocks: https://www.figma.com/file/H08AQJSaAIRxfeMP4iCUPh/UP-MVP?node-id=2022%3A4

A few things to note:
- Please use the BEM naming convention for styles: http://getbem.com/
- Prefer `rem` to `px` in styles

### PR DESCRIPTION
**Pull request by AlexK-ino**
## Description of the change

Added cards 
Added mock data to render these cards.

The greenish background and image of the card appear when the status is "EXCELLENT".

The rest, according to the task

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #871 

### Development

These boxes should be checked by the submitter prior to merging:

- [X] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/core/community/CoreVitalSummaryCard.scss [hunk 1]
```diff
@@ -0,0 +1,93 @@
+.CoreVitalSummaryCard {
+  max-width: 278px;
+  max-height: 96px;
+  width: 278px;
+  height: 96px;
```
Comment (Catacola) [inline src/core/community/CoreVitalSummaryCard.scss:5]: Please translate to rem throughout this file except for the shadows.

File: src/core/community/CoreVitalSummaryCard.scss [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      background-color: #a43939;
+      opacity: 0.7;
+    }
+    &.selected {
+      border: 1px solid #a43939;
+    }
+  }
+  &__poor {
+    &__top-border {
+      background-color: #a43939;
```
Comment (Catacola) [inline src/core/community/CoreVitalSummaryCard.scss:64]: Whoops, I forgot to give you the alpha channel. Please change this one and its border to `rgba(164, 57, 57, 0.7)`

File: src/core/community/CoreVitalSummaryCard.scss [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      opacity: 0.7;
+    }
+    &.selected {
+      border: 1px solid #006c67;
+    }
+  }
+  &__great {
+    background-color: #f7fffc;
+    &__percent {
+      background-image: url("./assets/img/wreath.svg");
```
Comment (Catacola) [inline src/core/community/CoreVitalSummaryCard.scss:83]: The wreath and the greenish background show display when percentage is 95% or more. This logic has it set to show when `status = "GREAT"`

File: src/core/community/CoreCommunityVitals.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    }
+    return 0;
+  };
+  const groupingCard = groupBy(mockCardContent, "status");
+  const orderedCard = [
+    ...get(groupingCard, "NEEDS_IMPROVEMENT", []).sort(sortFunction),
+    ...get(groupingCard, "POOR", []).sort(sortFunction),
+    ...get(groupingCard, "GOOD", []).sort(sortFunction),
+    ...get(groupingCard, "GREAT", []).sort(sortFunction),
+  ];
```
Comment (Catacola) [inline src/core/community/CoreCommunityVitals.js:59]: We don't need to sort the cards based on percentage or group. They just happened to be ordered that way in the mocks.

**Review by Catacola — CHANGES_REQUESTED** [review]
This is off to a good start! Besides a few nits below, the only other thing I noticed is that the text and wreath move by a pixel or two when the card becomes selected. For an extreme example which makes the text wrap, see the Timely Contacts card:
Unclicked:
Screen Shot 2021-03-12 at 1 23 28 AM
Clicked:
Screen Shot 2021-03-12 at 1 23 32 AM

Besides that, if you can please rebase against `feature/vitals` and clean up the git history so we only see your changes, it will make reviewing much easier. Thanks!

File: src/core/community/CoreVitalSummaryCard.scss (hunk 2 above)
Comment (AlexK-ino) [reply src/core/community/CoreVitalSummaryCard.scss:65]: Should this solution be in POOR status or NEEDS IMPROVEMENT?
image

File: src/core/community/CoreVitalSummaryCard.scss (hunk 3 above)
Comment (AlexK-ino) [reply src/core/community/CoreVitalSummaryCard.scss:84]: This style will only be used in a class that includes "great"

File: src/core/community/CoreCommunityVitals.js (hunk 4 above)
Comment (AlexK-ino) [reply src/core/community/CoreCommunityVitals.js]: Oh, I'm sorry. 
I don't fully understand how the data will come in, so I purposely mixed them up in the mock-data so they could be put in the right order at the front.
image
I'll just do them in this order then

File: src/core/community/CoreVitalSummaryCard.scss (hunk 2 above)
Comment (Catacola) [reply src/core/community/CoreVitalSummaryCard.scss]: You're right. It should be in "NEEDS IMPROVEMENT". Thanks for catching that.

File: src/core/community/CoreVitalSummaryCard.scss (hunk 3 above)
Comment (Catacola) [reply src/core/community/CoreVitalSummaryCard.scss:84]: The idea was the the wreath/green background should be independent of the dark green border used when status="GREAT". This way, we can have a card like the following that has the dark green border but no wreath:
Screen Shot 2021-03-12 at 9 40 30 AM

Let's modify the task slightly to add a 5th state: "EXCELLENT". The difference between "GREAT" and "EXCELLENT" is that they both have the dark green border, but "EXCELLENT" also has the green background and the wreath.

File: src/core/community/CoreVitalSummaryCard.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  </div>
+);
+CoreVitalSummaryCard.defaultProps = {
+  selected: false,
+};
+
+CoreVitalSummaryCard.propTypes = {
+  title: PropTypes.string.isRequired,
+  percentage: PropTypes.number.isRequired,
+  status: PropTypes.oneOf(["POOR", "NEEDS_IMPROVEMENT", "GOOD", "GREAT"])
```
Comment (Catacola) [inline src/core/community/CoreVitalSummaryCard.js:82]: \+ "EXCELLENT"

**Review by Catacola — APPROVED** [review]
Looks good! I'll make this one small change and land it. Thanks!

**Pull request merged**